testdata/stats/client_config_customized.yaml.tmpl

```diff
@@ -3,10 +3,15 @@ definitions:
 - name: requests_total
   value: "10"
   type: COUNTER
+- name: request_duration_milliseconds
+  value: "100"
+  type: COUNTER
 - name: custom
   value: "1"
   type: COUNTER
 metrics:
+  - name: request_duration_milliseconds
```

Comment:
I probably just don't understand the test but why does this show up on L6? It won't show up in /stats/promtheus I assume?

Reply:
good catch: leftover from early testing iterations. removed.